content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md

```diff
@@ -0,0 +1,67 @@
++++
+type = "news"
+title = "Media Release: The Haiku Project and Freewear launch Official Haiku Merchandise"
+author = "jt15s"
+date = "2021-12-01 12:17:33+11:00"
+tags = ["haiku", "software", "freewear", "merchandise", "donations"]
++++
+
+<p align="left">FOR IMMEDIATE RELEASE</p>
+<p align="left">December 01, 2021</p>
+
+# THE HAIKU PROJECT AND FREEWEAR LAUNCH OFFICIAL HAIKU MERCHANDISE
+The Haiku Project launched its official merchandise today in partnership with FreeWear, an online free and open-source software (FOSS) merchandising store and print shop. The partnership will allow The Haiku Project to offer a new way for people to financially contribute to the development of Haiku whilst getting a physical item in return for their gratitude. Donations greatly contribute to initiatives such as the recent development contract, which is allowing waddlesplash, a respected Haiku contributor, to make further general improvements to Haiku, such as system improvements, new and refined drivers and more. The partnership will mean that a portion of merchandise sales is donated to the Haiku Project, allowing The Project to continue funding both the development contract, pay for ongoing costs such as server hosting, and the chance to pursue other initiatives.   
+
+FreeWear has greatly simplified the process of selling merchandise, having built their own online store, and handling the printing and shipping processes as well as the process of donating a portion of the sales straight to the FOSS project/organisation. Moreover, most of the world can receive merchandise from FreeWear. A partnership with FreeWear does not involve any upfront costs from the FOSS project, as a portion of the sales pays from expenses incurred such as printing.   
```

Comment:
All of this is not really different from the previous store at cafepress.

The main changes for this new offer are:
- Transparency on the prices: it is clear how much money is donated to Haiku for each item
- Freewear also has partnerships with some events like Capitole du Libre, meaning people will be able to buy our merch in person in such events (sponsoring both the event and Haiku, and saving on shipping costs)